apps/backend/.eslintrc.js

```diff
@@ -14,13 +13,14 @@ module.exports = {
     jest: true
   },
   rules: {
+    'no-console': 'off',
+    '@typescript-eslint/no-explicit-any': 'error',
```

Comment:
we want to enforce and follow this going forward, avoid any unless there is an absolute need to have it. 

WIP and transitional work should still use types known at the time of the feature work.